packages/syft/src/syft/service/request/request.py

```diff
@@ -61,6 +61,20 @@ def is_type(self, type_: type) -> bool:
         return self.linked_obj and type_ == self.linked_obj.object_type
 
 
+@serializable()
+class ChangeStatus(SyftObject):
+    __canonical_name__ = "ChangeStatus"
+    __version__ = SYFT_OBJECT_VERSION_1
+
+    id: Optional[UID]
+    change_id: UID
+    applied: bool = False
+
+    @staticmethod
+    def from_change(change: Change, applied: bool) -> Self:
+        return ChangeStatus(change_id=change.id, applied=applied)
```

Comment:
```suggestion
    @classmethod
    def from_change(cls, change: Change, applied: bool) -> Self:
        return cls(change_id=change.id, applied=applied)
```

Not a big deal, just that if using classmethod then if someone subclass this it still works correctly.